pkg/storage/unified/sql/db/dbimpl/db.go

```diff
@@ -3,62 +3,31 @@ package dbimpl
 import (
 	"context"
 	"database/sql"
-	"fmt"
-	"strings"
 
 	"github.com/grafana/grafana/pkg/storage/unified/sql/db"
 )
 
+// NewDB converts a *sql.DB to a db.DB.
 func NewDB(d *sql.DB, driverName string) db.DB {
-	// remove the suffix from the instrumented driver created by the older
-	// Grafana code
-	driverName = strings.TrimSuffix(driverName, "WithHooks")
-
```

Comment:
We no longer need to account for the `WithHooks` drivers since those are the ones instrumented with the wrapper that was one of the causes of incident 2144. We will not see the `WithHooks` drivers because we return an informative [error](https://github.com/grafana/grafana/blob/fcfa4aa777dcd17836d81ea9dcf513bb95cc270f/pkg/storage/unified/sql/db/dbimpl/dbimpl.go#L27) if we find that case.